Makefile

```diff
@@ -75,7 +75,7 @@ fmt/buf:
 	buf format --write
 
 .PHONY: generate
-generate: generate/buf generate/examples
+generate: build generate/buf generate/examples
```

Comment:
I want the build task to run it automatically before running the generate task 🙏